samples/singleheader/amalgamate.cmake

```diff
@@ -9,8 +9,8 @@ function(amalgamate_ryml header_dir header_file)
         LIST_DIRECTORIES FALSE
         CONFIGURE_DEPENDS "${rymldir}/src")
     add_custom_command(OUTPUT "${singleheader}"
-        COMMAND python "${amscript}" "${singleheader}"
-        COMMENT "python ${amscript} ${singleheader}"
+        COMMAND python3 "${amscript}" "${singleheader}"
+        COMMENT "python3 ${amscript} ${singleheader}"
```

Comment:
The proper fix here is by calling [find_package(Python3 ...)](https://cmake.org/cmake/help/latest/module/FindPython3.html) and then using `${Python3_EXECUTABLE}` instead of `python` or `python3`.

Reply:
Much cleaner!